allauth/account/templatetags/account_tags.py

```diff
@@ -3,4 +3,4 @@
 warnings.warn("{% load account_tags %} is deprecated, use {% load account %}",
               DeprecationWarning)
 
-from account import *  # noqa
+from .account import *  # noqa
```

Comment:
@pennersr this (and a corresponding fragment below) is somehow not covered in unittests, but if you try this command in Python 3, it will fail, because implicit relative imports are not supported anymore:

```
from account.templatetags import account_tags
```